modules/nodes-nodes-working-deleting.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-managing.adoc
+
+[id='nodes-nodes-managing-deleting_{context}']
+= Deleting nodes from a cluster
+
+When you delete a node using the CLI, the node object is deleted in Kubernetes,
+but the pods that exist on the node are not deleted. Any bare pods not
+backed by a replication controller become inaccessible to {product-title}.
+Pods backed by replication controllers are rescheduled to other available
+nodes. You need to delete local manifest pods.
+
+.Procedure
+
+To delete a node from the {product-title} cluster:
+
```

Comment:
add node cordon first (with actually command example)

Reply:
Are you suggesting `oc adm cordon`? We don't mention this in the docs that I see on quick search.
Is this command different from `oc adm manage-node node1 --schedulable=false`?

```
. Mark the node as unschedulable:
+
----
oc adm cordon <node>
oc adm manage-node <node> --schedulable=false
----
+
For example:
+
----
oc adm cordon node1
oc adm manage-node node1 --schedulable=false

oc get nodes
NAME        STATUS                        ROLES     AGE       VERSION
node1     NotReady,SchedulingDisabled   compute   1d        v1.11.0+d4cacc0
----

```